test/common/http/conn_manager_impl_test.cc

```diff
@@ -4353,13 +4356,16 @@ TEST_F(ProxyStatusTest, PopulateProxyStatusWithDetailsAndResponseCode) {
   ASSERT_TRUE(altered_headers);
   ASSERT_TRUE(altered_headers->ProxyStatus());
   EXPECT_EQ(altered_headers->getProxyStatusValue(),
-            "custom_server_name; error=connection_timeout; details=\"bar; UT\"");
+            "custom_server_name; error=http_response_timeout; details=\"bar; UT\"");
```

Comment:
I think this test change is because of an error in the deprecation, and should probably be reverted.

Reply:
Sorry please disregard, was looking at the wrong PR